react/countries/ARG.tsx

```diff
@@ -0,0 +1,70 @@
+import { CountryRules } from '../typings/countryRulesTypes'
+
+const rules: CountryRules = {
+  fields: {
+    country: {
+      hidden: true,
```

Comment:
Will we still use this property? If it was used in the previous component to decide if the field was already completed and thus should be shown, then it should be deprecated for the new PlaceComponents.